grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package config;
+
+option go_package = "dataplane/grpc"
+
+service ConfigService {
+    rpc GetGlobalConfig (GetGlobalConfigRequest) returns (GlobalConfig);
```

Comment:
I think it's fairly obvious that config is global in that case so we can go with a short `GetConfig/UpdateConfig` and `Config`